ceci/stage.py

```diff
@@ -980,20 +1092,28 @@ def generate_command(cls, inputs, config, outputs):
 
         flags = [cls.name]
 
-        for tag, ftype in cls.inputs:
+        for tag, _ in cls.inputs:
+            if aliases is not None:
```

Comment:
Could write this as
`aliased_tag = tag if aliases is None else aliases.get(tag, tag)`
for clarity

Reply:
Or just do `aliases = aliases or {}` at the start of the function and then you can skip all the tests.

Reply:
Oh, that's a good idea.